uuid.go

```diff
@@ -255,3 +264,23 @@ func SetRand(r io.Reader) {
 	}
 	rander = r
 }
+
+// EnableRandPool enables internal randomness pool used for Random
+// (Version 4) UUID generation. The pool contains random bytes read from
+// the random number generator on demand in batches. Enabling the pool
+// may improve the UUID generation throughput significantly.
+//
+// Since the pool is stored on the Go heap, this feature may be a bad fit
+// for security sensitive applications.
+func EnableRandPool() {
```

Comment:
Over all I think this is fine, but there needs to be a comment that EnableRandPool and DisableRandPool are not thread-safe and should only be called when there is no possibility that New will be called concurrently.

Reply:
Good point. Improved the documentation in 9d84abfeeae2719feefa377b2628ac59758e6370